src/search/state/searchActions.ts

```diff
@@ -75,7 +75,7 @@ export const updateLogicOperator = (clauseId: string, value: Operator) =>
 
 export const submitAdvancedQuery = () => action(SUBMIT_ADVANCED_QUERY);
 
-export const addPreSelectedClauses = () => action(ADD_PRE_SELECTED_CLAUSES);
+export const addPreSelectedClauses = () => action(SET_PRE_SELECTED_CLAUSES);
```

Comment:
The action creators also follow the pattern of converting the action string constant to camelCase so `addPreSelectedClauses` would be `setPreSelectedClauses`

Reply:
Sorry, totally missed that. fixed now!